app/controllers/candidate_interface/content_controller.rb

```diff
@@ -18,12 +18,15 @@ def terms_candidate
       render_content_page :terms_candidate
     end
 
+    ProviderCourses = Struct.new(:provider_name, :courses)
```

Comment:
```suggestion
    provider_courses = Struct.new(:provider_name, :courses)
```

`UpperCamelCased` names are usually classes in Ruby!

Reply:
Also, this should be moved inside of `def providers`